dingo/gw/ls_cli.py

```diff
@@ -122,6 +124,20 @@ def ls():
                     sort_keys=False,
                 )
             )
+
+        elif dataset_type == "trained_model":
+            with h5py.File(args.file_name, "r") as f:
+                print(f"Dingo version: {f.attrs['version']}")
+                print("\nTrained model\n" + "================\n")
+
+                print(f"Approximant: {f.attrs['approximant']}")
+                print(f"Epoch: {f.attrs['epoch']}")
+
+                for d in ['model_kwargs', 'metadata']:
+                    json_data = json.loads(f['serialized_dicts'][d][()])
+                    print(f"\n{d}:\n" + "-"*(len(d)+1))
+                    pprint(json_data)
```

Comment:
For consistency, I think it's better to just print exactly what we do for a PT model but it's not a huge deal and this should be equivalent.

https://github.com/dingo-gw/dingo/blob/main/dingo/gw/ls_cli.py#L21-L33

Reply:
The last commit makes the output conform more closely to the one printed for `.pt` files. I left the `json` data alone, though.